vignettes/outbreak_distribution.Rmd

```diff
@@ -0,0 +1,318 @@
+---
+title: "Outbreak size and length distribution"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Outbreak size and length distribution}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>", 
+  fig.height = 5, 
+  fig.width = 8
+)
+```
+
+```{r setup}
+library(epichains)
+```
+
+In settings where there is no endemic transmission of an infectious disease it can be useful to understand the likelihood of different outbreak sizes. 
+
+The factors that influence outbreak size are the reproduction number of the infectious disease, the heterogeneity in transmission dynamics, population susceptibility, interventions and stochasticity in the epidemic.
+
+Using a branching process model we can seed a new epidemic with a single index case, and simulate the epidemic by defining an _offspring distribution_, which specifies the probability that each infected individual will go on to infect a further _N_ individuals (\( P(n = N) = p_N, N \in \{0, 1, 2, \ldots\} \)). The branching process simulation model can sample from the offspring distribution to generate a transmission chain, from which the outbreak size or outbreak length can be calculated. The branching process model is stochastic due to the random sampling of offspring distribution.
+
+::: {.alert .alert-primary}
+**Glossary**
+
+* Outbreak _size_: the total number of cases produced by an outbreak before it goes extinct.
+
+* Outbreak _length_: the total (maximum) number of generations reached by a outbreak before it goes extinct.
+:::
+
+In this vignette we will outline how to show the distribution of outbreak sizes and lengths using the _epichains_ R package. 
+
+## Simulating an outbreak size distribution
+
+First, we'll simulate the outbreak size distribution using the a Poisson offspring distribution. This is a simple one parameter distribution where the mean and variance of the distribution defined by $\lambda$ which is equal to the reproduction number ($R$).
+
+For this example we'll vary the reproduction number between 0.1 and 1.0, at 0.1 intervals.
+
+```{r, define-parameters}
+statistic <- "size"
+offspring_dist <- "rpois"
+R <- seq(0.1, 1.0, 0.1)
+```
+
+We will create a parameter space will each combination of parameters. In this simple outbreak size simulation, only the reproduction number varies. 
+
+```{r, parameter-grid}
+scenarios <- expand.grid(
+  offspring_dist = offspring_dist,
+  statistic = statistic,
+  R = R,
+  stringsAsFactors = FALSE
+)
+```
+
+The branching process is stochastic so we need to specify how many simulation model replicates to run to capture the stochasticity in the epidemic. This is important, as mentioned above as a factor that influences outbreak size, the same epidemic transmissibility parameters can result in different sized outbreaks depending on randomness, such as the number of infectees from a given infector.
```

Comment:
Branching processes are stochastic processes, so we need to specify how many simulation model replicates to run to capture the variability in the epidemic. This is important, as mentioned above as a factor that influences outbreak size, the same epidemic transmissibility parameters can result in different sized outbreaks depending on randomness, such as the number of infectees from a given infector.
```